src/SFA.DAS.AANHub.Database/Tables/Feedback.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Feedback]
+(
+    [AttendanceId] UNIQUEIDENTIFIER FOREIGN KEY REFERENCES [dbo].[Attendance](Id),

```

Comment:
I prefer having arbitrary primary key (Id) with compound unique index

Reply:
Just keeping it the same as Apprentice feedback and Employer Feedback - helps when referring to previous implementations.